client/src/components/classesPage/ClassListView.js

```diff
@@ -1,26 +1,114 @@
-import React from "react";
-import Grid from "@material-ui/core/Grid";
-import { withStyles } from "@material-ui/core/styles";
-import { NewClassCard, ExistingClassCard } from '../index.js'
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import { Grid, Card, Typography, Icon, CardContent } from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
+import { NewClassCard, ExistingClassCard } from '../index.js';
+import axios from 'axios';
 
 const styles = theme => ({
   wrapper: {
-    display: "flex",
-    justifyContent: "center",
-    alignItems: "center",
-    margin: "0 1rem"
+    display: 'flex',
+    height: '90vh',
+    margin: '0 1rem',
+    flexWrap: 'wrap',
+    border: '1px solid black'
+  },
+  card: {
+    margin: 20,
+    width: 200,
+    height: 200,
+    padding: theme.spacing.unit * 3,
+    display: 'flex',
+    border: '1px solid blue',
+    position: 'relative'
+  },
+  icon: {
+    margin: '0 auto',
+    position: 'absolute',
+    top: '50%',
+    left: '50%',
+    transform: 'translate(-50%, -40%)'
+  },
+  title: {
+    fontSize: 20,
+    fontWeight: 'bold',
+    marginBottom: 15
+  },
+  emptyList: {
+    display: 'flex',
+    alignItems: 'center',
+    flexDirection: 'column',
+    justifyContent: 'center'
   }
 });
 
 function ClassListView(props) {
+  const { classes } = props;
+  const [classList, setClassList] = useState([]);
 
-  return (
-    <Grid className={props.classes.wrapper}>
-      <ExistingClassCard />
-      <ExistingClassCard />
-      <NewClassCard />
-    </Grid>
-  );
-};
+  const ax = axios.create({
```

Comment:
I really like this abbreviation you've been using. Looks like a great variable name that is clear and saves a lot of typing.